Dockerfile

```
@@ -0,0 +1,16 @@
+FROM python:3.12-rc-alpine as base
+RUN apk update
+RUN apk add curl
```

Minor but you will sometimes see advice to combine layers where appropriate as it creates a slightly more optimised image, so e.g. here we might use:
```suggestion
RUN apk update && apk add curl
```

---

Changes made